stable/unifi/templates/discovery-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "unifi.fullname" . }}-discovery
+  labels:
+    app: {{ template "unifi.name" . }}
+    chart: {{ template "unifi.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.discoveryService.type }}
+  ports:
+    - port: {{ .Values.discoveryService.port }}
+      targetPort: http
```

Comment:
where is this named port defined?

Reply:
Hi @squat I'm not sure I follow the question.  The service `{{ template "unifi.fullname" . }}-discovery` has a port definition named `discovery` within the same spec: https://github.com/billimek/charts/blob/unifi/stable/unifi/templates/discovery-svc.yaml#L16

It's probably due to my lack of depth of knowledge around kubernetes services.  Can you elaborate on the question a bit more?

Reply:
The `targetPort` of a Service defines which port on the matched Pod should receive traffic. However, the unifi Pods have no port named `http`. It does have a port named `discovery`. This name should be used as the target instead IMO

Reply:
understood now, thank you for taking the time to explain.  Corrected the service definitions to match and be consistent.